pgex/examples/glow_effect/_types.py

```diff
@@ -0,0 +1,14 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+from typing import List, Sequence, Tuple, TypeAlias, Union
+
+import pygame
+
+Point: TypeAlias = Union[Tuple[int, int], List[int], pygame.Vector2, Sequence[int]]
+_RgbaOutput = Tuple[int, int, int, int]
+_ColorValue = Union[
```

Comment:
These two should also probably have a `TypeAlias` type hint. That hint's... I don't even know what's the point of it tbf, but basically it hints that the value is a type hint, but it's more about it being already used once here, so might as well add to all

Reply:
Since you are not even sure yourself about it, I guess I can share my opinion: typehints should be used wisely and putting them everywhere can at the end increase the complexity of the code.
Example:

```python
my_rect: pygame.Rect = pygame.Rect(position, size)
```
In this example, it is kinda obvious that we are manipulating a `pygame.Rect`, adding the typehint looks a bit redundant and at the end complexify the code.

Going back to our specific situation, from reading the code, we can easily understand that these variables are for type aliases, they are even defined in their own `_types` module.
So imo, typehint is useless in this context, unless you defined a strict policy about type hints everywhere because you want your IDE / linter to be able to catch/understand everything.